src/app/pages/dashboard/widgets/all-widgets.constant.ts

```diff
@@ -1,37 +1,47 @@
 import { WidgetType } from 'app/pages/dashboard/types/widget.interface';
 import { backupTasksWidget } from 'app/pages/dashboard/widgets/backup/widget-backup/widget-backup.definition';
 import { cpuWidget } from 'app/pages/dashboard/widgets/cpu/widget-cpu/widget-cpu.definition';
+import { arbitraryTextWidget } from 'app/pages/dashboard/widgets/custom/arbitrary-text/widget-arbitrary-text.definition';
 import { helpWidget } from 'app/pages/dashboard/widgets/help/widget-help/widget-help.definition';
 import { memoryWidget } from 'app/pages/dashboard/widgets/memory/widget-memory/widget-memory.definition';
+import { poolNameWidget } from 'app/pages/dashboard/widgets/memory/widget-pool-name/widget-pool-name.definition';
```

Comment:
`widget-pool-name` under memory folder.

Reply:
A `widget-storage` [coming soon](https://github.com/truenas/webui/pull/10040) and it uses `storage` folder